### PR TITLE
Remove compiler warnings

### PR DESCRIPTION
**Pull request by Framstag**

### CODE REVIEW

File: libosmscout-client-qt/src/osmscoutclientqt/RouteDescriptionBuilder.cpp [hunk 1]
```diff
@@ -69,23 +69,23 @@ static QString TurnCommandType(const RouteDescription::DirectionDescriptionRef&
 static QString ShortTurnCommand(const RouteDescription::DirectionDescriptionRef& directionDescription)
 {
   if (!directionDescription){
-    return osmscout::RouteDescriptionBuilder::tr("Turn");
```
Comment (Karry) [inline libosmscout-client-qt/src/osmscoutclientqt/RouteDescriptionBuilder.cpp:72]: I am worry that qt translation is not working with full namespace :-/ Will need to check

Comment (Framstag) [reply libosmscout-client-qt/src/osmscoutclientqt/RouteDescriptionBuilder.cpp:72]: OK, there ws a warning regarding namespaces, so I tried...

I'm still struggeling with glm+meson+OS X. Looks like I needed to make some updates, to get better C++20 support.

Comment (Framstag) [reply libosmscout-client-qt/src/osmscoutclientqt/RouteDescriptionBuilder.cpp:72]: See the gcc and cmake builds (still contain such wearnings)

Comment (Karry) [reply libosmscout-client-qt/src/osmscoutclientqt/RouteDescriptionBuilder.cpp:72]: this small comment fix lupdate: https://github.com/Framstag/libosmscout/pull/1598

**Pull request merged**